intercept/action.yaml

```diff
@@ -36,7 +36,6 @@ inputs:
   print_logs:
     description: This flag will determine if telepresence logs are printed out to the job's output and exported as an artifact.
     required: false
-    default: false
```

Comment:
the value passed to the action is a string repr of this value so we don't need this